### PR TITLE
Landing template for blog

### PR DESCRIPTION
**Pull request by baggerspion**
Issue: #141 

## Description

This PR contains all work related to strengthening the blog by adding a paginated landing page for the blog to be hosted at /blog/.

## Types of Changes

_What types of changes does your code introduce? remove the points which aren't applicable:_

- New feature (non-breaking change which adds functionality)

## Tasks

- [x] Add general template for landing pages
- [x] Add blog landing page at /blog/
  - [x] Separate first item
  - [x] Include title, image, author, post date, abstract
  - [x] Styling for first entry
  - [x] Styling for all other entries
- [x] Fix the "Blog" entry in the main menu
- [x] Rewrite "articles" to "blog" throughout
- [x] Update listing of blog posts on homepage to make it more interesting

### CODE REVIEW

**Comment by baggerspion** [comment]
OK, I think this is good to handover to you, @princiya.
The main work (to be done) of this PR is in blog/index.html. We basically need to apply the correct HTML+CSS to the elements I am already pulling in.

The first entry is separated out so that we can make it a *highlight* somehow: bigger image, longer abstract etc. All other items can then be "standard" listing.

Jekyll is handling pagination for us automatically. We will need to add some "dummy" content in order to look at (and style) the forward/back buttons etc.

**Comment by baggerspion** [comment]
Ready for review, @perploug

File: _includes/header.html [hunk 1]
```diff
@@ -1,31 +1,41 @@
...
+
       <li class="dc-tab__element dc-tab__element--header header__logo">
         <a href="/index.html">
           <img src="/assets/svg/logo.svg" />
         </a>
       </li>
-      {% if include.title == "Home" %}
-      <li class="dc-tab__element"><a href="#os-projects">Projects</a></li>
+
+      {% if page.permalink == "/" %}
```
Comment (perploug) [inline _includes/header.html:11]: This whole if/else thing for navigation is very confusion

**Review by perploug — APPROVED** [review]

File: _includes/header.html (hunk 1 above)
Comment (perploug) [reply _includes/header.html:11]: This is also worked on in the other docs branch - so will review the navigation after the merge and see it needs further refinement, it is not a must-do thing now  - the output is fine, just the code is a bit hard to read

**Pull request merged**